features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/Utils.js

```diff
@@ -354,6 +354,45 @@ class Utils {
             }, 100);
         }
     }
+
+    /**
+     * Force service definition download in browser
+     *
+     * @static
+     * @param {*} response
+     * @memberof Utils
+     */
+    static downloadServiceDefinition(response) {
```

Comment:
Can we use the existing forcee download util method to download service definitions too ?

Reply:
We can't use the existing one since we need few changes that cannot be done with single method